docker/Dockerfile.final

```diff
@@ -37,7 +37,7 @@ COPY --chown=macaron:macaron $WHEEL_PATH $HOME/dist/
 RUN : \
     && python3 -m venv $HOME/.venv \
     && . .venv/bin/activate \
-    && pip install --no-compile --no-cache-dir --upgrade pip \
+    && pip install --no-compile --no-cache-dir --upgrade pip setuptools \
```

Comment:
We don't need to install `setuptools` separately here.
```bash
python -m pip install --upgrade pip
```
already installs it:
```bash
$pip list
Package    Version
---------- -------
pip        23.2.1
setuptools 65.5.0
```

Reply:
I think we need to run it here with `--upgrade` to update it to the latest version though :thinking:

Reply:
That's a good point :+1: